packages/utils/src/cmds/transform_translate.js

```diff
@@ -0,0 +1,381 @@
+        fs.mkdirSync(argv.outDir, { recursive: true })
+    } else {
+        argv.verbose && log.debug('Output dir already exists, skip creating it')
+    }
+
+    const translationFiles = fs
+        .readdirSync(argv.inDir)
+        .filter(fileIsOldTranslationFile)
+
+    const mainTranslationFile = translationFiles.find(file =>
+        file.match(`i18n_module_${argv.primaryLanguage}.properties`)
+    )
+    const mainTranslationFilePath = path.join(argv.inDir, mainTranslationFile)
+
+    if (!fs.existsSync(mainTranslationFilePath)) {
+        log.error(
+            `Main language file must exist ("${path.join(
+                argv.inDir,
+                mainTranslationFile
+            )}")`
+        )
+        process.exit(1)
+    }
+
+    argv.verbose &&
+        log.debug('Extracting key/value pairs from translation files')
+    const translations = translationFiles.reduce((mappings, file) => {
+        const language = file.replace(/i18n_module_|.properties/g, '')
+        const contents = fs.readFileSync(path.join(argv.inDir, file), {
+            encoding: 'utf8',
+        })
+        const lines = contents
+            .split('\n')
+            .filter(line => line !== '' && !line[0].match(/\s*#/))
+
+        if (
+            languagesToTransform.length &&
+            // Primary language needed to build other translation files
+            language !== argv.primaryLanguage &&
+            languagesToTransform.indexOf(language) === -1
+        ) {
+            return mappings
+        }
+
+        mappings[language] = lines.reduce((mapping, line) => {
+            const [key, ...textParts] = line.split('=')
+            const text = textParts.join('=')
+
+            mapping[key] = text
+            return mapping
+        }, {})
+
+        return mappings
+    }, {})
+
+    for (const language in translations) {
+        if (translations.hasOwnProperty(language)) {
+            if (
+                languagesToTransform.length &&
+                languagesToTransform.indexOf(language) === -1
+            ) {
+                continue
+            }
+
+            const newLanguageFileName =
+                language === argv.primaryLanguage
+                    ? `${language}.pot`
+                    : `${language}.po`
+
+            let newContents = ''
+            const languageTranslations = translations[language]
+            const newLanguageFilePath = path.join(
+                argv.outDir,
+                newLanguageFileName
+            )
+            const shouldAppendContents =
+                !argv.overrideExistingFiles &&
+                fs.existsSync(newLanguageFilePath) &&
+                argv.appendToExistingFiles
+
+            if (
+                !argv.overrideExistingFiles &&
+                fs.existsSync(newLanguageFilePath) &&
+                !argv.appendToExistingFiles
+            ) {
+                console.log('')
+                log.debug(
```

Comment:
Many of these `log.debug` calls should probably be `log.info` (for highlighted things like indicating steps in a process) or `log.print` (for lower-priority but still not verbose things)